examples/runExample.ts

```diff
@@ -67,6 +67,7 @@ const repositoryLookupName = repositoryIdentifiers.map((identifier) => {
 });
 
 const finder = new mariner.IssueFinder(config);
+const contributorsFinder = new mariner.ContributorFetcher();
```

Comment:
I just realized that the local is named "Finder" but the function is named "Fetcher". They should be consistent. I think the outer thing should probably be a Finder in all cases, and the inner low-level thing should be a Fetcher. That would also reduce confusion between `contributorFetcher` and `gitHubContributorFetcher`.